README.md

```diff
@@ -19,85 +19,77 @@ MEGAGIG VET_CLINIC
   - [Run tests](#run-tests)
   - [Deployment](#triangular_flag_on_post-deployment)
```

Comment:
- [ ] @Megagig and @Ayokunnumi1 Great work implementing the changes requested, however the link for the tests and deployments still don't work, kindly consider fixing it or removing the links from your readme

Reply:
@danmainah This has been removed from the README. thanks